src/cli/command-processor.ts

```diff
@@ -0,0 +1,225 @@
+import {Worker} from 'worker_threads';
```

Comment:
the work interface seems generally useful to standardize the interface between the app and the platform w/ a message passing interface, did you consider adopting it for the library as well?

Reply:
I'm not as compelled to use it for the test runner interface since everything is pretty synchronous and the stubs/platform exist within the app already.  For implementing actual radio functionality I could see it helping.